sep490-frontend/src/app/modules/shared/components/dialog/confirm-dialog/confirm-dialog.component.ts

```
@@ -54,22 +56,22 @@
+    return this.option('acceptLabel') ?? 'TranslationKeys.ACCEPT';
   }
 
   get rejectButtonLabel(): string {
-    return this.option('rejectLabel') ?? "TranslationKeys.REJECT";
+    return this.option('rejectLabel') ?? 'TranslationKeys.REJECT';
   }
 
   option(name: string): any {
-    const source: { [key: string]: any } = this.confirmationOptions || this;
+    const source: {[key: string]: any} = this.confirmationOptions || this;
     if (source.hasOwnProperty(name)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)